src/main/java/Classes/Deadline.java

```diff
@@ -0,0 +1,15 @@
+package Classes;
+
+public class Deadline extends Task {
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
```

Comment:
Consider using a more descriptive variable name that better illustrates what the variable represents.